deploy/assisted-service.yaml

```diff
@@ -102,6 +102,8 @@ spec:
               value: "false"
             - name: LOG_LEVEL
               value: "info"
+            - name: INSTALL_RH_CA
+              value: "false"
```

Comment:
When will it be set to true?

Reply:
When the QE will need to test images from brew